Platforms/QemuQ35Pkg/PlatformPei/Platform.c

```diff
@@ -829,7 +829,8 @@ InitializePlatform (
 
     MmSettings.HeapGuardPolicy.Fields.MmPageGuard                    = 1;
     MmSettings.HeapGuardPolicy.Fields.MmPoolGuard                    = 1;
-    DxeSettings.ImageProtectionPolicy.Fields.ProtectImageFromUnknown = 1;
+    // Note: This is to leave a place holder for iPXE option rom...

```

Comment:
Cannot work around this for oprom... Open for other suggestions.

Reply:
IS this using E1000? I've generally found the virtio-net-pci setup better for UEFI features.